src/app/tests/suites/TestAccessControlConstraints.yaml

```diff
@@ -28,21 +28,36 @@ tests:
               - name: "nodeId"
                 value: nodeId
 
+    - label: "Read the fabric ID from the alpha fabric"
+      cluster: "Operational Credentials"
+      command: "readAttribute"
+      attribute: "CurrentFabricIndex"
+      response:
+          saveAs: alphaIndex
+
+    - label: "Read the commissioner node ID from the alpha fabric"
+      cluster: "CommissionerCommands"
+      command: "GetCommissionerNodeId"
+      response:
+          values:
+              - name: "nodeId"
+                saveAs: commissionerNodeIdAlpha
+
     - label: "Constraint error: PASE reserved for future (TC-ACL-2.4 step 29)"
       cluster: "Access Control"
       command: "writeAttribute"
       attribute: "ACL"
       arguments:
           value: [
                   {
-                      FabricIndex: 1,
+                      FabricIndex: alphaIndex,
```

Comment:
On write, this value is ignored, right?  Why are we trying to provide a "valid" value here?  I would much rather we provided 0.  Better yet would be if we could leave the value out completely in the YAML....

Reply:
This is correct. But for the moment this is needed by the codegen step at https://github.com/project-chip/connectedhomeip/blob/master/examples/darwin-framework-tool/templates/tests/partials/test_value.zapt#L22

I can probably fix that with an exception for 'FabricIndex' but it is unclear that it makes sense to spend time on that since I am going to remove this code soon ;)

Reply:
My point is: to placate the codegen step we can leave this in, but just use 0.  No need for alphaIndex.